gatsby-theme-marketing/src/scss/components/TroubleshootSpec.scss

```diff
@@ -13,12 +13,18 @@
   position: absolute;
 }
 
-.AbsoulteLintExpressions-wrapper {
+.LintExpressions-wrapper {
+  min-height: 44px;
```

Comment:
Does this also have a max-height? just want to make sure that as more errors show up in there and it grows that it does not get any taller than what is in the mocks